content/source/docs/cloud/users-teams-organizations/organizations.html.md

```diff
@@ -19,11 +19,11 @@ On most pages within Terraform Cloud, the top navigation bar displays the name o
 
 ![screenshot: the organization switcher menu](./images/org-nav.png)
 
-## Adding Users to Organizations
+## Joining and Leaving Organizations
 
-Organization membership is automatic, and is determined by team membership. To add a user to an organization, add them to one or more of that organization's teams. See [Teams][] for more information.
+To join an organization, you must be invited by an organization owner and must accept the emailed invitation. See [Organization Settings: Users](#users) below for details.
```

Comment:
> by an organization owner
suggested rephrase to: "to it by someone in the **owners** team of that organization"